app/src/utils/generateAPIKey.js

```diff
@@ -0,0 +1,10 @@
+import { nanoid } from 'nanoid';
+
+const generateAPIKey = () => {
+    const key = 'pfx_' + nanoid(); //pfx indicates that it's a key; default nanoid is 21 chars long
+    return key;
+}
+
+export default generateAPIKey;
+
+console.log(generateAPIKey());
```

Comment:
This is related to the comment above but I don't think this method is utilised anywhere else in the code.

Reply:
Used in `addAPIKey` method now.